testdata/gopath/src/project/pkg/webhook/default_server/namespace/mutating/update_webhook.go

```diff
@@ -17,8 +17,9 @@ limitations under the License.
 package mutating
 
 import (
-	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
 	corev1 "k8s.io/api/core/v1"
+
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
```

Comment:
Why do you split these into groups?

Reply:
I _think_ it's because corev1 is our target api in this case - interesting corner case!